File: CMOWebApi.Models/AdminModel/MasterModel/MenuClassificationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CMOWebApi.Models.AdminModel.MasterModel
{
	
	public class MenuClassificationModel
	{
		public int Id { get; set; }
		public Nullable<int> Code { get; set; }
		public string NameHindi { get; set; }
		public string NameEnglish { get; set; }
		public Nullable<int> DisplayOrder { get; set; }
		public string Remarks { get; set; }
		public Nullable<long> ClassificationType { get; set; }
		public bool IsActive { get; set; } = true;
		public bool IsDeleted { get; set; } = false;
		public Nullable<System.DateTime> CreatedDate { get; set; }
		public Nullable<int> CreatedBy { get; set; }
		public Nullable<System.DateTime> ModifiedDate { get; set; }
		public Nullable<int> ModifiedBy { get; set; }
		public List<string> MenuTypeMappingList { get; set; }
        public Nullable<bool> IsSubMenu { get; set; } = false;
        public Nullable<int> MenuClassificationCode { get; set; }
    }

}

File: ANGULAR/ANGULAR/src/app/Shared/Model/Master/menu-classification-model.ts
export class MenuClassificationModel {
		Id: number;
		Code: number | null;
		NameHindi: string;
		NameEnglish: string;
		DisplayOrder: number | null;
		Remarks: string;
		ClassificationType: number |string | null;
		IsActive: boolean;
		IsDeleted: boolean;
		CreatedDate: Date | string | null;
		CreatedBy: number | null;
		ModifiedDate: Date | string | null;
    ModifiedBy: number | null;
    MenuTypeMappingList: number[]|string[];
    IsSubMenu: boolean;
    MenuClassificationCode: number|string;
	}
